packages/tradershub/src/components/Modal/Modal.tsx

```diff
@@ -34,7 +34,7 @@ type TModal = {
  * @property {string} [className] - Optional CSS class name
  */
 export type TModalComponents = {
-    children: ReactNode;
+    children?: ReactNode;
     className?: string;
 };
```

Comment:
just a suggestion: you can make the children optional like this:
```suggestion
    className?: string;
};

//and then for the modal component:

const Modal = ({ children, className }: React.PropsWithChildren<TModal>) => {
 ....
}

```

Reply:
![image](https://github.com/binary-com/deriv-app/assets/100833613/ef4bf6f5-d22b-4d1f-9664-5a96d7885eb0)

Reply:
@shayan-deriv we want to specify the type for children of the modal(Header, Content and Footer) that's why PropsWithChildren will not be a perfect type here.